crates/trycp_server/test/test.js

```diff
@@ -124,3 +123,42 @@ sim2h_url = "wss://localhost:9001"
     })
     })
 }
+
+//doTestManager("ws://localhost:9000")
```

Comment:
Looks like some use of commenting to manually change the way the test runs. Would it make sense to put this behind an if block? Or is this not needed anymore?